src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/config/DomainConfiguration.kt

```diff
@@ -50,4 +52,10 @@ class DomainConfiguration {
     inductionEventService: InductionEventService,
   ): InductionService =
     InductionService(inductionPersistenceAdapter, inductionEventService)
+
+  @Bean
+  fun conversationDomainService(
+    conversationPersistenceAdapter: ConversationPersistenceAdapter,
+  ): ConversationService =
+    ConversationService(conversationPersistenceAdapter, null)
```

Comment:
As noted above, the `ConversationEventService` field on the `ConversationService` constructor is currently nullable, hence passing in `null` as the constructor arg here.

This will change in a few PR's time when we implement a `ConversationEventService` to register timeline events